### ISSUE TITLE
Adjecent 

### ISSUE THREAD

**Issue by BacchusFLT** [fadden/6502bench#118]
A c64 cart has multiple $2000 byte sized banks. So when a bank is mapped in, it is visible at $8000. If the data at the very start of a new bank is copied to another location, I guess this is a case where the regions fall short as I basically need to have two adjacent regions, and the program won't allow that.

The below should be $8000 as the start of the bank which is visible at $8000. The actual data is however a pseudopc block which will be copied to $0801. Knowing your c64, you can see the c64 basic header. One word for next basic line, one word for basic line number, $9e for SYS and then the SYS address as petscii characters.

                .addrs $8000
                .dd2  $080c           ;This is eventually copied to $0801
                .dd2  $000a
                .dd1  $9e
                .str  pet:“ 2064”
                .fill 5,$00

**Comment by fadden** [comment]
https://6502bench.com/sgmanual/editors.html#address

"You can create a child region that shares the same start offset as an existing region by selecting the first code or data line within that region. Note that regions with floating end points cannot have the same start offset as another region."

https://6502bench.com/sgmanual/intro-details.html#address-regions

"Regions that overlap must have a parent/child relationship. Whichever one starts last or ends first is the child. A strict ordering is necessary because a given file offset can only have one address, and if we don't know which region is the child we can't know which address to assign. Regions cannot straddle the start or end of another region, and cannot exactly overlap (have the same start and length) as another region. One consequence of these rules is that "floating" regions cannot share a start offset with another region, because their end point would be adjusted to match the end of the other region."

Bottom line: you can have two regions that start at the same place, but they can't also *end* at the same place.  If they overlapped totally, there wouldn't be any reason to declare the $8000 region, because no code would be assembled there.

**Comment by fadden** [comment]
FWIW, if you want to get a quick view of how the address map is laid out, use the new `Navigate > View Address Map` feature to get a listing.  I did it as text so it can be copy & pasted into bug reports.

**Comment by BacchusFLT** [comment]
I was indeed trying to have two floating ones. many thanks for the clarification and also many thanks for the advice on the Address Map. That was actually quite helpful.
